stdsimd/arch/detect/mod.rs

```diff
@@ -63,6 +63,13 @@ pub use self::arch::Feature;
 mod bit;
 mod cache;
 
+cfg_if! {
+    if #[cfg(all(target_arch = "aarch64", any(target_os = "freebsd")))] {
```

Comment:
Why is the `any(target_os = "freebsd")` needed here?  (shouldn't it just be `target_os = "freebsd"` ?)

Reply:
A subtle hint for potential future reuse of this module by other OSes :) I guess a comment would be better.

Reply:
Note that because we use `#[path]` here any os wanting to use the `aarch64` module can just use it by including it directly below - so there is no need to split the per-OS logic into two `cfg_if!`s.